File: adjacentElementsProduct.js
function adjacentElementsProduct(inputArray) {
    largestSum = inputArray[0] * inputArray[1];
    for (i = 1; i < inputArray.length - 1; i++){
        if (inputArray[i] * inputArray[i + 1] >= largestSum){
            largestSum = inputArray[i] * inputArray[i + 1];
        }
    }
    return largestSum;
}

File: intro_algorithms/adjacentElementsProduct/adjacentElementsProduct.py
def adjacentElementsProduct(inputArray):
    largest_sum = inputArray[0] * inputArray[1]
    for i in range(1, len(inputArray) - 1):
        if inputArray[i] * inputArray[i + 1] >= largest_sum:
            largest_sum = inputArray[i] * inputArray[i + 1]
    return largest_sum